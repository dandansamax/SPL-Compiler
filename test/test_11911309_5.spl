int test_1_r01(int a, int b)
{
  if (a > b)
  {
    return a;;
  }
  struct str{

  
  else
  {
    return b;
  }
}