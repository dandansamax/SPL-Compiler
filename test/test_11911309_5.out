Program (1)
  ExtDefList (1)
    ExtDef (1)
      Specifier (1)
        TYPE: int
      FunDec (1)
        ID: test_11911309_5
        LP
        VarList (1)
          ParamDec (1)
            Specifier (1)
              TYPE: int
            VarDec (1)
              ID: a
          COMMA
          VarList (1)
            ParamDec (1)
              Specifier (1)
                TYPE: int
              VarDec (1)
                ID: b
        RP
      CompSt (2)
        LC
        StmtList (3)
          Stmt (3)
            RETURN
            Exp (3)
              INT: 0
            SEMI
        RC
